tracer/test/test-applications/integrations/Samples.AWS.S3/SyncHelpers.cs

```diff
@@ -0,0 +1,97 @@
+#if NETFRAMEWORK
+
+using System;
+using System.IO;
+using System.Threading;
+using Amazon.S3;
+using Amazon.S3.Model;
+
+namespace Samples.AWS.S3
+{
+    static class SyncHelpers
+    {
+        private const string BucketName = "my-bucket";
+        private const string ObjectKey = "sample.txt";
+        private const string ObjectContent = "Hello World!";
+
+        public static void StartS3Tasks(AmazonS3Client s3Client)
```

Comment:
Should we also add coverage for the other new integrations like `CopyObject`, etc?